css/app/datepicker.css

```diff
@@ -71,14 +71,14 @@
 }
 
 .datepicker-heading .btn-arrow {
-	padding-bottom: 9px;
-	padding-top: 10px;
-	padding-left: 2px;
 	margin-right: 0;
+	height: 36px;
+	min-width: 25px;
+	padding: 5px;
 }
 
 .datepicker-heading .btn-date {
-	width: 180px;
+	width: 181px;
```

Comment:
Instead of defining the width, you could use flex :)
``` css
.datepicker-heading {
    margin: 0 5px;
    display: flex;
}
.datepicker-heading .btn-date {
    height: 36px;
    margin-left: 4px;
    flex-grow: 5;
}
```